templates/experiments/chat/input_bar.html

```diff
@@ -11,7 +11,7 @@
 
 <form class="w-full p-2 bg-gray-300"
       autocomplete="off"
-      hx-post="{% url 'experiments:experiment_session_message' request.team.slug experiment.id session.id %}"
+      hx-post="{% url 'experiments:experiment_session_message' request.team.slug experiment.id session.get_experiment_version_number session.id %}"
```

Comment:
FYI the tool referenced here is [ripgrep](@coderabbitai full review)

Reply:
I pushed a fix in commit 4b1dfd2bca6700a96a215f7ba2f9c6db71a4930d, please review it.

Reply:
@coderabbitai I pushed a fix in commit https://github.com/dimagi/open-chat-studio/commit/4b1dfd2bca6700a96a215f7ba2f9c6db71a4930d, please review it.

Reply:
So that was not an issue. The positional params alighned with what is expected in the url. All I did in the new commit was to assign the params to its respective keywords.